tests/plasma/test_grids.py

```diff
@@ -505,6 +506,24 @@ def uniform_cartesian_grid():
     return grid
 
 
+@pytest.mark.parametrize(
+    ("width"),
+    [
+        None,
+        0.1 * u.cm,
+    ],
+)
+def test_soften_edges(uniform_cartesian_grid, width):
+    grid = uniform_cartesian_grid
+    grid.soften_edges(width=width)
+    assert_quantity_allclose(grid["rho"][0, :, :], 0 * u.kg / u.m**3)
+    assert_quantity_allclose(grid["rho"][:, 0, :], 0 * u.kg / u.m**3)
+    assert_quantity_allclose(grid["rho"][:, :, 0], 0 * u.kg / u.m**3)
+    assert_quantity_allclose(grid["rho"][-1, :, :], 0 * u.kg / u.m**3)
+    assert_quantity_allclose(grid["rho"][:, -1, :], 0 * u.kg / u.m**3)
+    assert_quantity_allclose(grid["rho"][:, :, -1], 0 * u.kg / u.m**3)
```

Comment:
Looking back at this, would it be worthwhile to add one more test to make sure that the value at the center of the grid is very close to the original value there?

Reply:
Done!